pkg/exchange/max/exchange.go

```diff
@@ -311,6 +315,47 @@ func (e *Exchange) queryClosedOrdersByLastOrderID(ctx context.Context, symbol st
 	return types.SortOrdersAscending(orders), nil
 }
 
+func (e *Exchange) queryClosedOrdersByTime(ctx context.Context, symbol string, since, until time.Time) (orders []types.Order, err error) {
+	if err := e.closedOrderQueryLimiter.Wait(ctx); err != nil {
+		return orders, err
+	}
+
+	market := toLocalSymbol(symbol)
+	walletType := maxapi.WalletTypeSpot
+	if e.MarginSettings.IsMargin {
+		walletType = maxapi.WalletTypeMargin
+	}
+
+	req := e.v3client.NewGetWalletClosedOrdersRequest(walletType).Market(market).Limit(1000)
+
+	if !until.IsZero() {
+		req.Timestamp(until)
```

Comment:
what will happen if we don't set Timestamp?

Reply:
it will use now timestamp as default to query closed orders